test/tests/styleTest.js

```diff
@@ -12,7 +12,7 @@ describe("Zotero.Styles", function() {
 	});
 	
 	describe("Zotero.Styles.install", function() {
-		afterEach(function* (){
+		afterEach(`${styleID} style should be installed`, function* (){
```

Comment:
What? The string? Yeah, if the assertion fails in the hook it displays this string. Was reading mocha docs the other day and got surprised myself.